### ISSUE TITLE
Having a right-to-left language installed but not enabled causes style changes on a single table cell to apply to the whole row, if one or more cells have been split.

### ISSUE THREAD

**Issue by l0ss** [OfficeDev/office-js#4264]
## Your Environment
* Platform [PC desktop, Mac, iOS, Office on the web]: MacOS
* Host [Excel, Word, PowerPoint, etc.]: Word
* Office version number: 16.83
* Operating System: Sonomoa 14.3
 
## Expected behavior
When a single cell's style is changed via e.g. getCell(0,1).body.style, only that cell's style should be changed.

## Current behavior
If any cell in a row has been split, and a right-to-left language is installed in the OS (but not enabled), changing a single cell's style will cause all cells in the whole row to be set to that style.

## Steps to reproduce
1. Add a right-to-left language (e.g. Arabic/Hebrew) to the MacOS Input Sources menu, but do not enable it, and keep English input enabled.
2. Restart Word.
3. Create a 3x3 table in the document and set the styles of all cells to e.g. Normal
4. Split cell 0,1 to have two columns and 1 row, such that the table now has 4 columns in the top row, and 3 columns in the remaining rows
5. Use Script Lab or similar to run the following code:

```async function run() {
  await Word.run(async (context) => {
    const body = context.document.body;
    const tableCollection = body.tables;
    await context.sync();
    var findingTable = tableCollection.getFirst();
    await context.sync();
    findingTable.getCell(0, 1).body.style = "Body Text";
    await context.sync();
  });
}
```

6. Note that the entire first row has changed to "Body Text", rather than the expected single cell.

## Context
We have an add-in that automatically formats tables, and it works fine for most staff, but staff in middle eastern countries are forced to uninstall RTL languages from the OS entirely before using it.

**Comment by zhenhuangMSFT** [comment]
Hi @l0ss, thanks for reporting the issue! Does this also happen if you change the style from UI?

**Comment by l0ss** [comment]
No, using the UI doesn't trigger the bug, only using office.js to set the `body.style` attribute.

Another data point that may be useful: If I do something like this:

```
    findingTable.getCell(0, 1).body.select();
```

Then it ONLY selects cell 0,1, not the whole row.
Setting font size or colour via getCell(0,1).body.font seems to work fine too, so it does not appear to be a problem in `getCell()` or in getting the .body attribute of the cell, but specifically in setting the .style attribute on a single cell.

**Comment by Searion** [comment]
@l0ss 
Thanks for reporting this issue. 
It has been put on our backlog<Bug#8874592> for internal track. We will keep track of this issue and let you know if there are any updates.
